source/retryable-writes/tests/README.rst

```diff
@@ -0,0 +1,126 @@
+=====================
+Retryable Write Tests
+=====================
+
+The YAML and JSON files in this directory tree are platform-independent tests
+that drivers can use to prove their conformance to the Retryable Writes spec.
+
+Converting to JSON
+==================
+
+The tests are written in YAML because it is easier for humans to write and read,
+and because YAML includes a standard comment format. A JSONified version of each
+YAML file is included in this repository. Whenever a YAML file is modified, the
+corresponding JSON file should be regenerated. One method to convert to JSON is
+using `yamljs <https://www.npmjs.com/package/yamljs>`_::
+
+    npm install -g yamljs
+    yaml2json -s -p -r .
+
+Version
+=======
+
+Files in the "specifications" repository have no version scheme. They are not
+tied to a MongoDB server version, and it is our intention that each
+specification moves from "draft" to "final" with no further revisions; it is
+superseded by a future spec, not revised.
+
+However, implementers must have stable sets of tests to target. As test files
+evolve they will occasionally be tagged like "retryable-write-tests-YYYY-MM-DD",
+until the spec is final.
+
+Server Fail Point
+=================
+
+The tests depend on a server fail point, ``disconnectAfterWrite``, which allows
+us to force a network error after the server executes a write command but before
+it would return a write result. The fail point may be configured like so::
+
+    db.runCommand({
+        configureFailPoint: "disconnectAfterWrite",
+        mode: { times: <integer> },
+        data: { errorOnFirstAttemptOnly: <boolean> }
+    });
```

Comment:
Might want to mention that in case of failure the cleanup of each test should disable the failpoint via `mode: "off"` so as not to cause spurious test failures in subsequent tests.